modules/installation-mirror-repository.adoc

```diff
@@ -30,38 +30,84 @@ link:https://access.redhat.com/downloads/content/290/[{product-title} downloads
 
 . Set the required environment variables:
+.. Export the release version:
 +
+[source,terminal]
 ----
-$ export OCP_RELEASE=<release_version> <1>
-$ export LOCAL_REGISTRY='<local_registry_host_name>:<local_registry_host_port>' <2>
-$ export LOCAL_REPOSITORY='<local_repository_name>' <3>
-$ export PRODUCT_REPO='openshift-release-dev' <4>
-$ export LOCAL_SECRET_JSON='<path_to_pull_secret>' <5>
-$ export RELEASE_NAME="ocp-release" <6>
-$ export ARCHITECTURE=<server_architecture> <7>
-$ REMOVABLE_MEDIA_PATH=<path> <8>
+$ export OCP_RELEASE=<release_version>
 ----
-<1> For `<release_version>`, specify the tag that corresponds to the version of {product-title} to
++
+For `<release_version>`, specify the tag that corresponds to the version of {product-title} to
 install, such as `4.5.4`.
-<2> For `<local_registry_host_name>`, specify the registry domain name for your mirror
+
+.. Export the local registry name and host port:
++
+[source,terminal]
+----
+$ export LOCAL_REGISTRY='<local_registry_host_name>:<local_registry_host_port>'
+----
++
+For `<local_registry_host_name>`, specify the registry domain name for your mirror
 repository, and for `<local_registry_host_port>`, specify the port that it
 serves content on.
-<3> For `<local_repository_name>`, specify the name of the repository to create in your
+
+.. Export the local repository name:
++
+[source,terminal]
+----
+$ export LOCAL_REPOSITORY='<local_repository_name>'
+----
++
+For `<local_repository_name>`, specify the name of the repository to create in your
 registry, such as `ocp4/openshift4`.
-<4> The repository to mirror. For a production release, you must specify
-`openshift-release-dev`.
-<5> For `<path_to_pull_secret>`, specify the absolute path to and file name of
-the pull secret for your mirror registry that you created.
-<6> The release mirror. For a production release, you must specify
-`ocp-release`.
-<7> For `server_architecture`, specify the architecture of the server, such as `x86_64`.
-<8> For `<path>`, specify the path to the directory to host the mirrored images.
+
+.. Export the name of the repository to mirror:
++
+[source,terminal]
+----
+$ export PRODUCT_REPO='openshift-release-dev'
+----
++
+For a production release, you must specify `openshift-release-dev`.
+
+.. Export the path to your registry pull secret:
++
+[source,terminal]
+----
+$ export LOCAL_SECRET_JSON='<path_to_pull_secret>'
+----
++
+For `<path_to_pull_secret>`, specify the absolute path to and file name of the pull secret for your mirror registry that you created.
+
+.. Export the release mirror:
++
+[source,terminal]
+----
+$ export RELEASE_NAME="ocp-release"
+----
++
+For a production release, you must specify `ocp-release`.
+
+.. Export the type of architecture for your server, such as `x86_64`.:
++
+[source,terminal]
+----
+$ export ARCHITECTURE=<server_architecture>
+----
+
+.. Export the path to the directory to host the mirrored images:
++
+[source,terminal]
+----
+$ REMOVABLE_MEDIA_PATH=<path>
```

Comment:
Is there a reason why this one doesn't start with "export"? I think it should be:

`$ export REMOVABLE_MEDIA_PATH=<path>`

Reply:
@wking, is there a reason that we're not exporting the flash drive path when you mirror install/update content?

Reply:
I still think we should only `export` variables that need to be consumed by sub-processes, and not the rest of these local placeholders (#22008).

Reply:
I agree. That makes sense. I just wasn't sure why the one variable was different than the others.